lib/pure/random.nim

```diff
@@ -656,4 +656,22 @@ when not defined(nimscript) and not defined(standalone):
       let now = times.getTime()
       randomize(convert(Seconds, Nanoseconds, now.toUnix) + now.nanosecond)
 
+const cb64safe = {
+  'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+  'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
+  'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
+  'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
+  '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '-', '_',
+} # Copied from base64.nim URL-Safe to be compatible.
+
+proc randToken*(buffer: var openArray[char]; alphabet = cb64safe) {.inline, since: (1, 5).} =
+  ## Generate a random ASCII URL-Safe `buffer` containing `alphabet` chars.
+  # Inspired but not copied from Python 3.10 "secrets.token_urlsafe()".
```

Comment:
Another difference: the procs is not cryptographically secure and we should notify users.

Reply:
Already documented on bold at top

https://github.com/nim-lang/Nim/blob/c548f97241c9f0da8f9442c07b99dd9d6adf225a/lib/pure/random.nim#L17

For the whole module.

Reply:
But I agree on it too, I mean maybe a PR to change **All bold notes** to the new cool "Admonitions" thingy would ne nice.
:)